docs/running-on-yarn.md

```diff
@@ -105,6 +105,13 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
   </td>
 </tr>
 <tr>
+ <td><code>spark.executor.instances</code></td>
+  <td>2</td>
+  <td>
+    The number of executors.
```

Comment:
Can this clarify that spark.executor.instances is ignored when using dynamic allocation?